client/src/components/application/NameExamination.vue

```diff
@@ -17,7 +17,7 @@
 
       <!-- msg re. in progress with someone else -->
       <div class="alert alert-warning examiner-warning"
-           v-if="currentState == 'INPROGRESS' && examiner != userId && auth">
+           v-if="userCanEdit && (currentState == 'INPROGRESS') && (examiner != userId) && auth">
```

Comment:
This is just a message re. who owns the NR. I don't agree that we should hide it for non-editors, I think it should be displayed for everyone as it's useful info.